main/ui/src/main/java/org/cryptomator/ui/controllers/ChangePasswordController.java

```diff
@@ -68,12 +78,29 @@ public ChangePasswordController(Application app, Localization localization) {
 	@FXML
 	private Hyperlink downloadsPageLink;
 
+	@FXML
+	private Label passwordStrengthLabel;
+
+	@FXML
+	private Rectangle passwordStrengthShape;
```

Comment:
I'm not yet sure, if we use just one rect or multiple segments - but this is just UI polishing. for this PR keep the rect ;)